actionpack/lib/action_controller/metal/conditional_get.rb

```diff
@@ -77,18 +77,16 @@ def etag(&etagger)
     #
     #   before_action { fresh_when @article, template: 'widgets/show' }
     #
-    def fresh_when(record_or_options, additional_options = {})
-      if record_or_options.is_a? Hash
```

Comment:
Gotcha, for some reason in my mind that'd only work with ** not individual declarations, :+1: